charts/victoria-metrics-anomaly/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 - TODO
 
+## 1.0.0
+
+**Release date:** 2024-02-01
+
+![AppVersion: v1.9.2](https://img.shields.io/static/v1?label=AppVersion&message=v1.9.2&color=success&logo=)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+- Add ability to render vmanomaly config as is via `vmanomalyConfig`.
```

Comment:
Hey, change notes should be under `Next release` and new version title will be generated automatically.
As the [release CI](https://github.com/VictoriaMetrics/helm-charts/blob/master/.github/workflows/release.yaml#L56) reads content under `Next release` and determine whether to release new version.
```
## Next release

- Add ability to render vmanomaly config as is via `vmanomalyConfig`.
- xxx
```